### ISSUE TITLE
Rule proposal: `vue/no-bind-string-literal`

### ISSUE THREAD

**Issue by FloEdelmann** [vuejs/eslint-plugin-vue#1623]
**Please describe what the rule should do:**

This rule forbids passing a string literal to a prop with `v-bind` in the template.

**What category should the rule belong to?**
[x] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<!-- BAD -->
<comp :foo="'bar'" />
<comp :foo='"bar"' />
<comp :foo="`bar`" />

<!-- GOOD -->
<comp foo="bar" />
<comp :foo="'ba' + 'r'" />
<comp :foo="`ba${'r'}`" />
```

This rule should be auto-fixable.

**Comment by FloEdelmann** [comment]
Sorry, I didn't search well enough. This rule already exists: [`vue/no-useless-v-bind`](https://eslint.vuejs.org/rules/no-useless-v-bind.html#vue-no-useless-v-bind).
